proposals/load_geojson.json

```diff
@@ -0,0 +1,51 @@
+{
+    "id": "load_geojson",
+    "summary": "Load GeoJSON data",
+    "description": "Loads GeoJSON data as defined by [RFC 7946](https://www.rfc-editor.org/rfc/rfc7946.html) into a vector data cube. Feature properties are preserved.",
```

Comment:
```suggestion
    "description": "Loads inline GeoJSON data as defined by [RFC 7946](https://www.rfc-editor.org/rfc/rfc7946.html) into a vector data cube. Feature properties are preserved.",
```

Reply:
I'm wondering whether load is the right term. We should probably stick to it in the id, but maybe in the description we can say something like "Processes inline GeoJSON" or so?

Reply:
Updated